package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clarity-js",
-  "version": "0.1.16",
+  "version": "0.2.0",
```

Comment:
That's a jump... ambitious pull request :)

Reply:
I moved id up because there is a data format change

---
In reply to: [144364548](https://github.com/Microsoft/clarity-js/pull/39#discussion_r144364548) [](ancestors = 144364548)

Reply:
I agree this is an appropriate bump. We will probably also go through multiple version on the "main" 0.1 series before this one goes out too. 

---
In reply to: [144371478](https://github.com/Microsoft/clarity-js/pull/39#discussion_r144371478) [](ancestors = 144371478,144364548)